README.md

```diff
@@ -1094,6 +1095,89 @@ Server: spray-can/1.3.3
 }
 ```
 
+## GET /workflows/:version/:id/metadata
```

Comment:
I said that's fine :)

Previously it was something like call-metadata but in a few days that'll be a misnomer. I was just suggesting putting an appropriate name in now, which I think this is